portable-atomic-util/build.rs

```diff
@@ -35,6 +35,8 @@ fn main() {
         println!(
             "cargo:rustc-check-cfg=cfg(portable_atomic_no_alloc,portable_atomic_no_alloc_layout_extras,portable_atomic_no_core_unwind_safe,portable_atomic_no_error_in_core,portable_atomic_no_futures_api,portable_atomic_no_io_safety,portable_atomic_no_io_vec,portable_atomic_no_maybe_uninit,portable_atomic_no_min_const_generics,portable_atomic_no_track_caller,portable_atomic_no_unsafe_op_in_unsafe_fn,portable_atomic_sanitize_thread)"
         );
+        // unstable feature cfg
+        println!("cargo:rustc-check-cfg=cfg(portable_atomic_unstable_coerce_unsized)")
```

Comment:
Please add to the top-level Cargo.toml as suggested in https://github.com/taiki-e/portable-atomic/pull/195#discussion_r1850479122.

(Workspace lints will be propagated properly: https://docs.rs/crate/portable-atomic-util/latest/source/Cargo.toml#136)